package.json

```diff
@@ -120,6 +122,8 @@
   "devDependencies": {
     "ava": "^1.0.0-beta.0",
     "babel-eslint": "^8.2.3",
+    "coveralls": "^3.0.3",
+    "nyc": "^13.3.0",
```

Comment:
We probably need to get a quick +1 from security for the new deps

Reply:
true

Reply:
https://github.com/brave/internal/issues/520